src/components/familyInfo.js

```diff
@@ -84,18 +82,7 @@ export default function FamilyInfo ({ nextUrl }) {
           defaultValue={prevFormData.contactEmail}
           errors={errors}
         />
-
-        <Box display="flex" justifyContent="flex-start" paddingTop={2} paddingBottom={1}>
-          <Box display="flex" justifyContent="flex-start" paddingRight={2}>
-            <Button component={Link} to="w2jobinfo" variant="contained" color="secondary" >
-              Back
-            </Button>
-          </Box>
-
-          <Button type="submit" variant="contained" color="primary">
-            Save and Continue
-          </Button>
-        </Box>
+        {children}
```

Comment:
what does children represent here?

Reply:
Same as any react component that can have a children property, just children of the `<FamilyInfo>` element. I settled on doing it this way instead of including some parameter (`isLastPage` or something like that), so that all the page usages in `Main.tsx` would specify their own buttons as children. Is it good?

Reply:
Better in my opinion would be to have a <FormWizard /> element, which steps through the children routes, but I couldn't figure out a clean way to have the <form> element in each page, and the submit button injected into each page's form.

Reply:
> Same as any react component that can have a children property, just children of the `<FamilyInfo>` element. I settled on doing it this way instead of including some parameter (`isLastPage` or something like that), so that all the page usages in `Main.tsx` would specify their own buttons as children. Is it good?

Could you change the variable name from children to navButtons for clarity?

Reply:
Sure.

Reply:
Done